messagebus/src/main/java/com/yahoo/messagebus/routing/Resender.java

```diff
@@ -98,10 +99,11 @@ boolean scheduleRetry(RoutingNode node) {
      */
     public void resendScheduled() {
         List<RoutingNode> sendList;
+
+        long now = SystemTimer.INSTANCE.milliTime();
```

Comment:
Downside is that we will now always take the time even if the queue is empty, but not likely to be an issue in practice.

Reply:
The rationale is that I think the cost in the ballpark of synchronized, but it is better done on the outside. It is not a big thing, but I prefer to make the synchonized section as short as possible.